torch/csrc/inductor/aoti_eager/kernel_meta_info.h

```diff
@@ -41,9 +41,11 @@ struct TensorMetadata {
   // Sizes of a tensor. Currently, we only support static shape and use int64_t
   // to represent the sizes. In the future, we will create symbolic size and use
   // SymInt to represent it to support symbolic shape.
-  std::vector<int64_t> sizes_;
+  std::vector<std::optional<c10::SymInt>> sizes_;
```

Comment:
What does the `std::optional` mean here? When does the value not exist?

Reply:
The `TensorCheck` requires the type. So, I refined it a little bit to align with `TensorCheck`